### ISSUE TITLE
Documentation on CircleCI usage with multiple Dockerfiles suggests commas rather than colons

### ISSUE THREAD

**Issue by asheppard-gresham** [hadolint/hadolint#669]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
Hadolint should run on CircleCI against multiple Dockerfiles, comma separated, as per: https://github.com/hadolint/hadolint/blob/master/docs/INTEGRATION.md#circleci

### Actual behavior
String separation is based on colons rather than commas, so produces errors that Dockerfiles cannot be found: `engine/Dockerfile,`. Note the comma on the end, producing an invalid filepath.

### Steps to reproduce the behavior
CircleCI config:
```
orbs:
  docker: circleci/docker@1.6.0

workflows:
  lint:
    jobs:
      - docker/hadolint:
          dockerfiles: 'api/Dockerfile, engine/Dockerfile, ui/Dockerfile'
```

Build output:
```bash
DOCKERFILES=api/Dockerfile, engine/Dockerfile, ui/Dockerfile

# use colon delimiters to create array
arrDOCKERFILES=(${DOCKERFILES//:/ })
let END=${#arrDOCKERFILES[@]}

for ((i=0;i<END;i++)); do
  DOCKERFILE="${arrDOCKERFILES[i]}"

  hadolint $DOCKERFILE

  echo "Success! $DOCKERFILE linted; no issues found"
done

Running hadolint with the following options...


/bin/bash: line 6: engine/Dockerfile,: No such file or directory
```

**Additional environment details (OS, stack version, etc.)**
